Apps/W1/HybridGP/app/src/Migration/Support/GPConfiguration.table.al

```diff
@@ -33,6 +33,9 @@ table 4024 "GP Configuration"
         {
             DataClassification = SystemMetadata;
             InitValue = false;
+            ObsoleteState = Pending;
```

Comment:
Same comment regarding if not CLEAN tags

Reply:
Could this cause future upgrades to fail since it would be actually removing the field?

Reply:
Thanks